Userland/Libraries/LibWebView/InspectorClient.cpp

```diff
@@ -505,6 +510,16 @@ void InspectorClient::context_menu_delete_all_cookies()
     m_cookie_context_menu_index.clear();
 }
 
+void InspectorClient::set_is_windowed(bool is_windowed)
+{
+    if (!m_inspector_loaded)
+        return;
+
+    auto const* is_visible_string = is_windowed ? "false" : "true";
+    auto const script = MUST(String::formatted("inspector.setCloseInspectorButtonVisibility({});"sv, is_visible_string));
```

Comment:
A couple things: 

1. Don't put "sv" at the end of format string literals. This prevents compile-time validation of the format string.
2. Booleans are formatted as "true" / "false", so you can pass the boolean directly.

So something like:
```c++
 auto script = MUST(String::formatted("inspector.setCloseInspectorButtonVisibility({});", !is_windowed));
```